changelog.md

```diff
@@ -2,6 +2,23 @@
 
 ## 2.6.0~rc1
 
+
+
+## 2.5.2
+
+### Web applications
+- Updated dependencies:
+  - werkzeug from 2.0.2 to 2.2.3, markupsafe from 2.0.1 to 2.1.2, Flask-SQLAlchemy from 2.4.0
+ to 2.5.1 (#6752)
+  - cryptography from 3.4.7 to 39.0.1 (#6746, #6751)
+  - certifi from 2017.7.27.1 to 2022-12.07 (#6708)
+  - flask from 2.0.2 to 2.0.3 (#6624,#6634)
```

Comment:
I didn't want to hold up #6758 for this tiniest of nits, but here I will.  @zenmonkeykstop, let me know by the end of the day today if you have any objections to breaking the strict backport-ness of this pull request.  :-)

```suggestion
  - flask from 2.0.2 to 2.0.3 (#6624, #6634)
```

Reply:
No issues, go for it.